language/src/main.rs

```diff
@@ -185,8 +195,36 @@ impl Callbacks for HacspecCallbacks {
                         .err("unknown backend extension for output file");
                     return Compilation::Stop;
                 }
+                };
+                
+                match &self.output_template_file {
+                    None => if self.copy_template {
+                        Command::new("cp")
```

Comment:
Unfortunately this might not work on Windows machines and is generally a bit iffy to use.
To copy files, there's https://doc.rust-lang.org/std/fs/fn.copy.html